Watchman.DomainModel/Help/Models/HelpInformation.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using Watchman.Integrations.MongoDB;
+
+namespace Watchman.DomainModel.Help.Models
```

Comment:
```suggestion
namespace Watchman.DomainModel.HelpInformations
```
To obiekt domenowy a nie model